pkg/util/httputil/client.go

```diff
@@ -0,0 +1,52 @@
+package httputil
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"net/http"
+	"net/url"
+
+	"os"
+
+	"github.com/authgear/authgear-server/pkg/lib/config"
```

Comment:
util cannot import lib. Create a new struct in this package instead.

Reply:
Moved http client inside `authn/sso`